generators/addons/index.js

```diff
@@ -111,6 +117,23 @@ module.exports = class extends Generator {
         // }
     }
 
+    _addContinuousConfig() {
+        let adoFileName = 'azure-pipelines.yml';
+        let karmaFileDestPath = 'config/karma.config.js';
+        let karmaFileSourcePath = 'karma.config.js';
+        if (!fs.existsSync(this.destinationPath(adoFileName))) {
```

Comment:
You do not need to check if the file already exists, because you just deploy it from the generator to the target. If it already exists yeoman asks you what to do. In this case, it shouldn't be there at all.
Same with file below.
Yeoman handles this through use of a virtual filesystem before it gets written to disk.

Reply:
Already removed it.